drivers/overlay/ov_serf.go

```diff
@@ -83,7 +83,8 @@ func (d *driver) notifyEvent(event ovNotify) {
 	n := d.network(event.nid)
 	ep := n.endpoint(event.eid)
 
-	ePayload := fmt.Sprintf("%s %s %s", event.action, ep.addr.IP.String(), ep.mac.String())
+	ePayload := fmt.Sprintf("%s %s %s %s", event.action, ep.addr.IP.String(),
```

Comment:
Again I think we probably can send `ep.addr.String()` which will also send the mask and then parse it back to an `IPNet` value in the receiving side rather than sending the `IP` and `Mask` separately.